src/main/kotlin/com/github/julianhille/sqldelight/jscipherdriver/JsSqlCipherDriver.kt

```diff
@@ -0,0 +1,207 @@
+package com.github.julianhille.sqldelight.drivers.jscipher
+
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.db.SqlPreparedStatement
+import com.squareup.sqldelight.Transacter
+import org.khronos.webgl.Uint8Array
+import org.khronos.webgl.Int8Array
+
+
+@JsNonModule @JsModule("better-sqlite3-multiple-ciphers")
+external fun createDatabase(path: String? = definedExternally, options: DatabaseOptions? = definedExternally): Database
+
+class DatabaseConfiguration(
+  val name: String,
+  var path: String,
+  var schema: SqlDriver.Schema,
+  var create: (SqlDriver) -> Unit,
+  var upgrade: (SqlDriver, Int, Int) -> Unit,
+  var journalMode: Boolean = true,
+  val memory: Boolean = false,
+  var key: String? = null,
+) {
+
+  fun dbPath(): String {
+    if (memory) return ":memory:"
+    return path + name
+  }
+}
+
+class SqlJsCipherDriver (var configuration: DatabaseConfiguration ): SqlDriver {
```

Comment:
SQLDelight class is called

`JsSqlDriver`

So more consistent would be

```suggestion
class JsSqlCipherDriver (var configuration: DatabaseConfiguration ): SqlDriver {
```

or any variation, keeping `JsSql` together

Reply:
Actually, the file name already is called `JsSqlCipherDriver`

Reply:
And same should go with other class names as well. 
e.g. `SqlJsCipherStatement` -> `JsSqlCipherStatement`